Packs/Gatewatcher/.secrets-ignore

```diff
@@ -0,0 +1,2 @@
+2.5.3.102
+20.20.20.20
```

Comment:
Please verify that those aren't real secrets - we don't want to expose the real data of customers or users.

Reply:
The first entry is our GCenter appliance version.
The second entry is just an arbitrary IP address used in unit tests.

Reply:
Ok good :)